docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,365 @@
+{
```

Comment:
<p>In terms of describing the benefit, we can be more concrete. Specifically:</p><p> </p><p>With <code>rx</code> gates, all single-qubit rotations (<code>u3</code>) can be achieved with a single control pulse, rather than being decomposed into two <code>sx</code> gates and three <code>rz</code> gates. This reduces the duration and error for single-qubit gates by up to a factor of two.</p><p> </p><p>Similarly, with <code>rzz</code> gates, this avoids a decomposition into multiple <code>cz</code> gates for a common operation in time evolution circuits.</p><p> </p><p>Also, can we please also show an example of using <code>use_fractional_gates</code> with <code>service.backend(...)</code>? The <code>least_busy(...)</code> method is also not guaranteed to return a Heron processor, which is necessary for these examples to work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>